app/controllers/search_controller.rb

```diff
@@ -17,15 +19,17 @@ class SearchController < ApplicationController
     per_page
   ].freeze
 
-  FEED_PARAMS = %i[
-    page
-    per_page
-    published_at
-    search_fields
-    sort_by
-    tag_names
-    user_id
-    class_name
+  FEED_PARAMS = [
+    :class_name,
+    :page,
+    :per_page,
+    :search_fields,
+    :sort_by,
+    :user_id,
+    {
+      tag_names: [],
+      published_at: [:gte]
```

Comment:
What's the `[:gte]`?

Reply:
"greater than or equal to" it is how we perform [a range query in Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-range-query.html#range-query-ex-request)

Reply:
Ohh interesting. TIL, thanks!